recipes/libjxl/meta.yaml

```diff
@@ -110,8 +110,7 @@ outputs:
       commands:
         - cjxl --version
         - djxl --version
-        - cjpeg_hdr --version
```

Comment:
Isnt this a useful executable?

Reply:
Yes, it is. But it does not have a `--version` option. I'm making another test